FinniversKit/Sources/Components/TextField/TextField.swift

```diff
@@ -183,7 +187,7 @@ public class TextField: UIView {
         case .phoneNumber:
             isValidByInputType = isValidPhoneNumber(text)
         case .normal, .multiline:
-            isValidByInputType = true
+            isValidByInputType = isValidRegEx(text)
```

Comment:
Could we move the implementation of the method here? 🙂 The name of the method is somewhat misleading since it seems it checks the validity of the regex and not if the text matches it.

I.e.
```swift
if let textRegex = textRegex else {
    isValidByInputType = evaluate(textRegex, with: text)
} else {
    isValidByInputType = true
}
```